pages/apim/3.x/installation-guide/docker/installation-guide-docker-introduction.adoc

```diff
@@ -5,18 +5,68 @@
 :page-layout: apim3x
 :page-description: Gravitee.io API Management - Installation Guide - Docker - Introduction
 :page-keywords: Gravitee.io, API Management, apim, guide, manual, docker, linux
-:page-toc: false
 :page-liquid:
+:page-toc: false
+
+Our documentation describes six different ways you can install APIM using Docker.
+
+* link:{{ '/apim/3.x/apim_installation_guide_docker_compose_quickstart.html' | relative_url }}[Quick install with `docker compose`]
+* link:{{ '/apim/3.x/apim_installation_guide_docker_compose_quickstart_ee.html' | relative_url }}[Quick install with `docker compose` (Enterprise Edition)]
+* link:{{ '/apim/3.x/apim_installation_guide_docker_compose.html' | relative_url }}[Custom install with `docker compose`]
+* link:{{ '/apim/3.x/apim_installation_guide_docker_compose_ee.html' | relative_url }}[Custom install with `docker compose` (Enterprise Edition)]
+* link:{{ '/apim/3.x/apim_installation_guide_docker_images.html' | relative_url }}[Docker images install]
+* link:{{ '/apim/3.x/apim_installation_guide_docker_images_ee.html' | relative_url }}[Docker images install (Enterprise Edition)]
 
-// author: Tom Geudens
+You should be familiar with Docker before installing with Docker.

```

Comment:
Consider adding a quick overview for the intended audience as it pertains to quick install vs custom install so users understand what they can gain by completing the custom install

Reply:
Done